q2_metadata/plugin_setup.py

```diff
@@ -59,3 +64,86 @@
                 'visualization supports interactive filtering, sorting, and '
                 'exporting to common file formats.',
 )
+
+ArtificialGrouping = \
+    SemanticType('ArtificialGrouping', variant_of=SampleData.field['type'])
```

Comment:
Is this definitely sample data after all? It seems like you could use this to permute taxonomic labels or other kinds of metadata axes as well.

Reply:
I suppose it could be used for feature metadata too. I'm not exactly sure how to make that generalization, so I'd need some input there (and it's unlikely that I could get it done in the next week due to other commitments). I'm open to the change if someone has the time to get it in in time for the release, or we could create an issue to keep it in mind for the future. (I _think_ the action could be generalized in a backward compatible way in the future as none of the interface components imply samples.)

Reply:
That's fine, I was just pondering. SampleData is sufficient for now.